app/resources/numismatic_artists/numismatic_artists_controller.rb

```diff
@@ -17,9 +18,18 @@ def after_create_success(obj, change_set)
     super
   end
 
+  def after_delete_success
+    flash[:alert] = "Artist was deleted successfully"
+    redirect_to solr_document_path(@coin)
+  end
+
   private
 
     def parent_resource
       @parent_resource ||= find_resource(@change_set.artist_parent_id)
     end
+
+    def load_coin
```

Comment:
We eventually want to use this for linking to artists from both Coins and NumismaticIssues, right? Do you want to change the name to `load_parent` or something more neutral?

Reply:
I can update to load_parent